share/completions/krita.fish

```diff
@@ -12,17 +7,17 @@ function __krita_complete_image_format
             case '*,*,*'
                 # nothing is completed as arbitrary width and height are expected
             case '*,'
-                __krita_append_image_format_value 'U8
+                echo 'U8
```

Comment:
No need for the `echo` and awkward line-breaks.

Just give it as arguments to `string join ,`, like I explained.

Reply:
This code doesn't work as expected:

```fish
function __krita_complete_image_format
    set -l previous_token (commandline -oc)[-1]
    set -l current_token (commandline -t)

    if test "$previous_token" = --new-image
        switch $current_token
            case '*,*,*'
                # nothing is completed as arbitrary width and height are expected
            case '*,'
                string join , $current_token U8 U16 F16 F32
            case '*'
                echo 'RGBA
XYZA
LABA
CMYKA
GRAY
YCbCrA' | string replace -r '$' ','
        end
    end
end
```

Completions for U8 U16 F16 F32 are broken: when I press tab having say XYZA already typed I get `krita --new-image GRAY,,U8,U16,F16,F32` instead of list of choices.

Reply:
Ah, hold on, you're expecting output like

```
XYZA,
LABA,
CMYKA,
```

That's easier as

```fish
printf '%s,\n' XYZA LABA CMYKA # ...
```

which also gets us away from the awkward linebreaks and echo-to-thing piping.

Reply:
Fixed. ;)